toolkits/jira/arcade_jira/tools/sprint_planning.py

```diff
@@ -0,0 +1,410 @@
+import logging
+from datetime import datetime
+from typing import Annotated, Any, cast
+
+from arcade_tdk import ToolContext, tool
+from arcade_tdk.auth import Atlassian
+from arcade_tdk.errors import ToolExecutionError
+
+from arcade_jira.client import JiraClient
+from arcade_jira.constants import BOARD_TYPES_WITH_SPRINTS
+from arcade_jira.tools.boards import get_boards
+from arcade_jira.utils import (
+    build_sprint_params,
+    clean_sprint_dict,
+    create_error_entry,
+    create_sprint_result_dict,
+    validate_sprint_limit,
+)
+
+logger = logging.getLogger(__name__)
+
+# Error message constants
+BOARD_IDS_REQUIRED_ERROR = (
+    "Board IDs are required. You must specify scrum boards to fetch sprint data."
+)
+CONFLICTING_DATE_PARAMS_ERROR = (
+    "Cannot use 'specific_date' together with 'start_date' or 'end_date' parameters. "
+    "Please use either specific_date alone or start_date/end_date for date range filtering."
+)
+
+
+@tool(
+    requires_auth=Atlassian(
+        scopes=[
+            "read:board-scope:jira-software",
+            "read:project:jira",
+            "read:sprint:jira-software",
+            "read:issue-details:jira",
+            "read:board-scope.admin:jira-software",
```

Comment:
Can you add comments on the right side of each scope specifying what the scope is needed for (which endpoints). This is especially useful when a tool calls other tools or when a tool calls multiple endpoints. We haven't done a very good job at documenting this type of thing in the past, but I think it would be great if we started doing this by default.